l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -114,8 +114,9 @@ def initialize(
 
         self.root.attrs["format_version"] = FORMAT_VERSION
         self.root.attrs["labels"] = PERCEPTION_LABELS
+        return self
```

Comment:
Return `self` is strange

Reply:
it's quite common for fluent API interfaces I think. I would leave it as it is, it doesn't have any side effect and avoid us 1 line around all the codebase